imports/ui/pages/components/Modaler/GameListModal.js

```diff
@@ -54,63 +55,54 @@ const GameListModal = ({ gameList, isImported, history, onClose, ...rest }) => {
+              dataSource={formattedGameList}
+              pagination={{ position: ["none", "bottomRight"] }}
+              onRow={row => ({
+                onClick: () => {
+                  handleSetExaminMode(row.id);
+                  onClose();
+                }
+              })}
             >
-              <thead>
-                <tr>
-                  <th style={{ textAlign: "center", background: "#f1f1f1", padding: "5px 5px" }}>
-                    {translate("players")}
-                  </th>
-                  <th style={{ textAlign: "center", background: "#f1f1f1", padding: "5px 5px" }}>
-                    {translate("result")}
-                  </th>
-                  <th style={{ textAlign: "center", background: "#f1f1f1", padding: "5px 5px" }}>
-                    {translate("date")}
-                  </th>
-                  <th style={{ textAlign: "center", background: "#f1f1f1", padding: "5px 5px" }}>
-                    {translate("pgn")}
-                  </th>
-                </tr>
-              </thead>
-              <tbody>
-                {formattedGameList.map((game, index) => (
-                  <tr key={index} style={{ cursor: "pointer" }}>
-                    <td
-                      style={{ padding: "5px 5px" }}
-                      onClick={() => {
-                        handleSetExaminMode(game.id);
-                        onClose();
-                      }}
-                    >
-                      {game.white}-vs-{game.black}
-                    </td>
-                    <td style={{ padding: "5px 5px" }}>{game.result}</td>
-                    <td style={{ padding: "5px 5px" }}>{game.time}</td>
-
-                    <td style={{ padding: "5px 5px" }}>
-                      <a href={"export/pgn/history/" + game.id} className="pgnbtn">
-                        <img
-                          src="images/pgnicon.png"
-                          style={{ width: "25px", height: "25px" }}
-                          alt="PgnDownload"
-                        />
-                      </a>
-                    </td>
-                  </tr>
-                ))}
-              </tbody>
-            </table>
+              <Table.Column
+                title={translate("players")}
+                key="players"
+                render={(text, record) =>
+                  translate("playersColumn", {
+                    white: record.white,
+                    black: record.black
+                  })
+                }
+              />
+              <Table.Column title={translate("result")} dataIndex="result" key="result" />
+              <Table.Column title={translate("date")} dataIndex="time" key="time" />
+              <Table.Column
+                title={translate("pgn")}
+                dataIndex="pgn"
+                key="pgn"
+                render={(text, record) => (
+                  <a href={"export/pgn/history/" + record.id} className="pgnbtn">
```

Comment:
Do not use global styles. Create `Button` component that will incapsulate those styles